terracumber_config/tf_files/SUSEManager-Test-Ion-NUE.tf

```diff
@@ -134,38 +135,38 @@ module "cucumber_testsuite" {
       provider_settings = {
         mac = "aa:b2:93:01:00:42"
       }
-      additional_packages = [ "venv-salt-minion" ]
-      install_salt_bundle = true
+//      additional_packages = [ "venv-salt-minion" ]
+//      install_salt_bundle = true
     }
     suse-client = {
-      image = "sles15sp4o"
+      image = "sles15sp3o"
```

Comment:
Not sure if we test with SP3 for 4.2. If we do, then this is OK :)

Reply:
Same, just mimic what we have at https://github.com/SUSE/susemanager-ci/blob/master/terracumber_config/tf_files/SUSEManager-4.2-NUE.tf

Reply:
ACK.